src/main/java/teammates/ui/template/FeedbackResponseCommentRow.java

```diff
@@ -0,0 +1,38 @@
+package teammates.ui.template;
+
+import teammates.common.datatransfer.FeedbackResponseCommentAttributes;
+
+public class FeedbackResponseCommentRow {
+    private String giverDetails;
+    private FeedbackResponseCommentAttributes comment;
+    private String creationTime;
+    private ElementTag editButton;
+
+    public FeedbackResponseCommentRow(String giverDetails, FeedbackResponseCommentAttributes comment,
+                                         String creationTime, ElementTag editButton) {
+        this.giverDetails = giverDetails;
+        this.comment = comment;
+        this.creationTime = creationTime;
+        this.editButton = editButton;
+    }
+
+    public String getGiverDetails() {
+        return giverDetails;
+    }
+
+    public FeedbackResponseCommentAttributes getComment() {
+        return comment;
+    }
+    
+    public String getCommentText() {
+        return comment.commentText.getValue();
+    }
```

Comment:
Is a significant number of the variables contained in `comment` used on the frontend?
Would it be better to extract them into independent `String`s and such?